### DISCUSSION TITLE
Release Proposal for Apache Kvrocks 2.10.1

### DISCUSSION THREAD

**Discussion by git-hulk** [apache/kvrocks#2609] [General]
We got feedback from one user who mentioned that requests were blocked for a long time after upgrading to 2.10.0, please refer to #2596. And we confirmed it will affect all users when the compaction happens. We also have fixed a few severe issues in the stream group and JSON mset command. So we feel it'd be better to have a patch version release 2.10.1.   

I propose to bring the following commits into 2.10.1:

- 149d4fbd
- 290012ec
- 3b8dcf6e
- ff32f95b
- 94eb31cf
- 0d5f851
- 313a1d9

For the cherry-pick branch, see: https://github.com/apache/kvrocks/commits/2.10/

**Comment by git-hulk** [comment]
@apache/kvrocks-committers To see if you guys have other suggestions? I will send a new release vote call in a few days if no objection.

**Comment by PragmaTwice** [comment]
0d5f8514b2f09f6d512fd17ae8d5a3ec57935eda should also be included.

The rest looks good.

**Reply by git-hulk** [reply]
Added, thanks for replenishing.

**Comment by git-hulk** [comment]
The vote thread: https://lists.apache.org/thread/h291d7333kh5yh1cg0sb3253vm3hs1mf.

Welcome to verify and vote.
